### ISSUE TITLE
Android Intent Filter can't set property AutoVerify

### ISSUE THREAD

**Issue by martintreurnicht** [expo/expo-cli#83]
## Description

When trying to set android `autoVerify` setting on `intentFilters` you get an error.

## Expected Behavior

You should be able to set `autoVerify` to `true`, the code has been there for a while

## Observed Behavior

When running `expo-cli build:android` you get the following error:

```
[11:08:04] Error: Problems validating fields in app.json. See https://docs.expo.io/versions/v30.0.0/guides/configuration.html.
[11:08:04]  • Field: android.intentFilters[0] - should NOT have additional property 'autoVerify'.
[11:08:04] Couldn't publish because errors were found. (See logs above.) Please fix the errors and try again.
```

## Environment

Environment:
  OS: macOS 10.14
  Node: 10.7.0
  Yarn: 1.7.0
  npm: 6.1.0
  Watchman: 4.9.0
  Xcode: Xcode 10.0 Build version 10A255
  Android Studio: Not Found

Packages: (wanted => installed)
  expo: ^30.0.0 => 30.0.1
  react: 16.3.1 => 16.3.1
  react-native: https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz => 0.55.4

**Comment by martintreurnicht** [comment]
https://github.com/expo/expo-cli/pull/82 Fixes this issue

**Comment by fson** [comment]
Fixed in #82.
